docs/static/configuration.asciidoc

```diff
@@ -510,18 +510,23 @@ doesn't exist versus a field that's simply false. The expression `if [foo]` retu
 
 For more complex examples, see <<using-conditionals, Using Conditionals>>.
 
-[float]
+NOTE: Date math in conditionals is not currently supported. 
```

Comment:
Maybe more appropriate as `sprintf time format` - `Date math` is more typically an Elasticsearch term